wrapper.py

```diff
@@ -310,8 +310,9 @@ def _load_custom_shape(f):
     """
     data_reshaped = []
     counter = 0
```

Comment:
while you are at it, could you remove the manual counter and use an `enumerate` in the loop?
```python
for counter, l in enumerate(f.attrs['oldshape']):
```